jsk_kinova_robot/README.md

```diff
@@ -69,9 +69,10 @@ mkdir -p ~/kinova_ws/src
 cd ~/kinova_ws/src
 wstool init
 wstool merge https://raw.githubusercontent.com/jsk-ros-pkg/jsk_robot/master/jsk_kinova_robot/kinova.rosinstall
+wstool merge https://raw.githubusercontent.com/Kanazawanaoaki/jsk_robot/kinova-noetic/jsk_kinova_robot/kinova.rosinstall.${ROS_DISTRO} # Please ignore error if kinova.rosinstall.${ROS_DISTRO} does not exist. # We can remove this line after the https://github.com/ros/collada_urdf/pull/44 is merged
 wstool update
 cd ../
-source /opt/ros/melodic/setup.bash
+source /opt/ros/${ROS_DISTRO}/setup.bash
```

Comment:
NITS: The environment variable `${ROS_DISTRO}` has not been set unless the user source `/opt/ros/.../setup.bash` . So you had better writing like `/opt/ros/noetic/setup.bash # If noetic`

Reply:
I put `source /opt/ros/*/setup.bash` first because .rosinstall part also uses distro. https://github.com/jsk-ros-pkg/jsk_robot/pull/1906/commits/6193b0fa54d98c0170c46de58a4a5f9c269a6c06